ziti/cmd/edge/quickstart.go

```diff
@@ -303,69 +310,67 @@ func (o *QuickstartOpts) run(ctx context.Context) {
 }
 
 func (o *QuickstartOpts) createMinimalPki() {
-	if !o.AlreadyInitialized {
```

Comment:
The rest of the changes below this point are a reduction of tab indent by one level as a result of eliminating the redundant conditional `if not initialized`. This was removed because createMinimalPki() is now invoked only after internally evaluating whether dbDir exists, and when it does not exist then o.AlreadyInitialized is false and we proceed to create PKI.